apps/cookbook/src/app/showcase/header-showcase/header-showcase.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component } from '@angular/core';
+import {
+  ApiDescriptionProperty,
+  ApiDescriptionPropertyColumns,
+} from '~/app/shared/api-description/api-description-properties/api-description-properties.component';
+
+@Component({
+  selector: 'cookbook-header-showcase',
+  templateUrl: './header-showcase.component.html',
+  styleUrls: ['./header-showcase.component.scss'],
+  preserveWhitespaces: true,
+})
+export class HeaderShowcaseComponent {
+  properties: ApiDescriptionProperty[] = [
+    {
+      name: 'title',
+      description: 'A string that is used as the title.',
+      type: ['string'],
+      defaultValue: '',
+    },
+    {
+      name: 'value',
+      description: 'A string that is used as the value.',
+      type: ['string'],
+      defaultValue: '',
+    },
+    {
+      name: 'subtitle1',
+      description: 'A string that is used as the first subtitle.',
+      type: ['string'],
+      defaultValue: '',
+    },
+    {
+      name: 'subtitle2',
+      description: 'A string that is used as the second subtitle.',
+      type: ['string'],
+      defaultValue: '',
+    },
+    {
+      name: 'centered',
+      description: 'A boolean that describe whether the header should be centered.',
+      type: ['boolean'],
+      defaultValue: 'false',
+    },
+    {
+      name: 'titleMaxLines',
+      description:
+        'A number that describes that allowed amount of lines in the title before truncating the value.',
+      type: ['number'],
+      defaultValue: '2',
+    },
+  ];
+
+  directiveColumns: ApiDescriptionPropertyColumns = {
+    name: 'Name',
+    description: 'Description',
+  };
+
+  directives: ApiDescriptionProperty[] = [
+    {
+      name: 'kirbyHeaderCustomTitle',
+      description:
+        'The `kirbyHeaderCustomTitle` directive can be applied to an element as an alternative to passing the title property. Using this method you will gain more control over the content.',
+    },
+    {
+      name: 'kirbyHeaderCustomValue',
+      description:
+        'The `kirbyHeaderCustomValue` directive can be applied to an element as an alternative to passing the value property. Using this method you will gain more control over the content.',
+    },
```

Comment:
Maybe we should consider removing `kirbyHeaderCustomValue` directive for now and add it later, when we know the custom scenarios better? According to @henrikvoetmand 9 out of 10 times the scenario is adding a unit to the `value` e.g. `USD` in which case we should handle the formatting according to the design specs:
![image](https://user-images.githubusercontent.com/9556693/222371278-abfa23e0-6b86-485a-9f6f-13c62a8bac09.png)